ubccr/xdmod
# Fix Role Delegation

## Description
Updated the `setOrganizations` section of both `update_user.php` and `create_user.php`. The way the incoming `acl` data was updated in a previous PR meant that these two blocks of code were not being executed. This led to user_acl_group_by_parameters and UserRoleParameters not being in sync with user_acls and UserRoles. Which meant that code that solely depended on these tables for information would not function properly. In general this was not noticed as anyplace that these tables were being referenced there were additional sources of information used as a fallback. Unfortunately one of the places that did not have a fallback was the code that populated the organization name displayed on the users profile ( `CenterDirectorRole::getFormalName` -> `CenterDirectorRole::getActiveCenter` ). 

**NOTE:** This did not affect what data a user was authorized to view as that code has been migrated from utilizing these tables.

## Motivation and Context
So that we display consistent and correct information to users.

## Tests performed
Manual tests were performed:
  - Create a new User w/ some combination of 'cd', 'cs'
  - Log in as the newly created user
  - Click the 'My Profile' button
  - Ensure that the 'Top Role' value properly displays the most privileged role + the name of the center they are associated with.

  - Update an existing user to have some combination of 'cd', 'cs'
  - Log in as the updated user
  - Click the 'My Profile' button
  - Ensure that the 'Top Role' value properly displays the most privileged role + the name of the center the user is associated with.

## Types of changes


## Checklist:
